packages/web/src/new/utils/analytics/amplitude.ts

```diff
@@ -0,0 +1,28 @@
+import { init, track } from '@amplitude/analytics-browser';
+import assert from 'ts-invariant';
+
+import type { AmplitudeActions, ComplexActions } from 'new/utils/analytics/types';
+
+export const initAmplitude = () => {
+  assert(
+    process.env.REACT_APP_AMPLITUDE_API_KEY,
+    "REACT_APP_AMPLITUDE_API_KEY doesn't set in environment",
+  );
+  init(process.env.REACT_APP_AMPLITUDE_API_KEY);
+};
+
+export const trackEvent1 = (action: AmplitudeActions) => {
+  if ((action as ComplexActions).params) {
```

Comment:
Do we need this condition? The `track` method cannot accept `undefined` in second argument?

Reply:
if i see it can - https://github.com/amplitude/Amplitude-TypeScript/blob/6d2486ae86bbac5b202440e3468e4540fcfe7c6e/packages/analytics-core/src/core-client.ts#L72

Reply:
yes. it can. but simple actions do not have `params`. So wee need to do something like `track(action.name,  (action as ComplexActions).params)` in order not have TS errors. I think such code will look a little bit strange

Reply:
we had type `MapActionWithNoData` which works perfect before and gave us ability to don't use `ComplexActions` just to write useless condition